test/luajit-test-init.lua

```diff
@@ -100,6 +100,45 @@ rawset(_G, 'pairs', pairs_M.builtin_pairs)
 assert(pairs ~= nil)
 assert(type(pairs) == 'function')
 
+-- Remove the override loader.
+--
+-- If the override loader is enabled at least one test in the
+-- LuaJIT submodule fails: PUC-Rio-Lua-5.1-tests/attrib.lua.
+--
+-- A sketchy description of the test case of the question is the
+-- following.
+--
+-- There is a directory:
+--
+--  | + libs/
+--  | +- foo.lua
+--  | +- bar.lua
+--
+-- And the Lua code:
+--
+--  | package.path = 'libs/?.lua;libs/bar.lua'
+--  | local foo = require('foo')
+--
+-- The test case expects that `foo` contains a return value of
+-- libs/foo.lua.
+--
+-- However when the override loader is enabled, the following
```

Comment:
Typo: s/However/However,/

Reply:
Fixed. Resolved the discussion as trivial.